lib/matplotlib/axes/_secondary_axes.py

```diff
@@ -123,21 +124,12 @@ def apply_aspect(self, position=None):
         self._set_lims()
         super().apply_aspect(position)
 
-    def set_ticks(self, ticks, *, minor=False):
-        """
-        Set the x ticks with list of *ticks*
-
-        Parameters
-        ----------
-        ticks : list
-            List of x-axis tick locations.
-        minor : bool, default: False
-            If ``False`` sets major ticks, if ``True`` sets minor ticks.
-        """
-        ret = self._axis.set_ticks(ticks, minor=minor)
+    def set_ticks(self, ticks, labels=None, *, minor=False, **kwargs):
+        ret = self._axis.set_ticks(ticks, labels, minor=minor, **kwargs)
         self.stale = True
         self._ticks_set = True
         return ret
+    set_ticks.__doc__ = Axis.set_ticks.__doc__
```

Comment:
I think that's `@docstring.copy(Axis.set_ticks)`?

Reply:
Yes.